src/modules/ekf2/ekf2_main.cpp

```diff
@@ -1121,9 +1121,15 @@ void Ekf2::run()
 				flow.flowdata(0) = optical_flow.pixel_flow_x_integral;
 				flow.flowdata(1) = optical_flow.pixel_flow_y_integral;
 				flow.quality = optical_flow.quality;
+#if 0
```

Comment:
I think this shouldn't be needed because in the optical flow fusion in the ecl library we already check if flow.gyrodata is equal to NAN and if it is we take the gyro data from the onboard imu. https://github.com/PX4/ecl/blob/572710ed275551df088955b146235e27896198c4/EKF/optflow_fusion.cpp#L551